packages/lesswrong/components/common/Header.tsx

```diff
@@ -483,6 +567,13 @@ const Header = ({
                 </div>
               </Typography>
               {!isEAForum &&<ActiveDialogues />}
+              {isGivingSeason &&
```

Comment:
Currently this is shown on all pages on desktop (and no pages on mobile). I'm not sure if we want this or if it should just be on the home page.